root/static/scripts/common/components/TagEditor.js

```diff
@@ -558,18 +553,22 @@ export const MainTagEditor = (hydrate<TagEditorProps>(
 
           {(positiveTagsOnly && !tags.every(isAlwaysVisible)) ? (
             <>
-              {this.props.$c.user?.has_confirmed_email_address ? (
-                <p>
-                  {l(
-                    `Tags with a score of zero or below,
-                     and tags that you’ve downvoted are hidden.`,
-                  )}
-                </p>
-              ) : (
-                <p>
-                  {l('Tags with a score of zero or below are hidden.')}
-                </p>
-              )}
+              <SanitizedCatalystContext.Consumer>
```

Comment:
Why add several `Consumer` here rather than assigning `$c` once?

Reply:
These create components that subscribe to context changes.  So the difference would be the entire component re-rendering when the context changes vs. just the sections that use it.  We don't currently change the `$c` context at all, but maybe we will in the future, and I think it's bad form to code the app in such a way that context changes would degrade performance.

(If you mean why not use `useContext`, you can't use hooks inside class-based components.)

Reply:
Since all the Consumer components are adjacent I merged those together at least -- the diff is nicer if you enable "Hide whitespace"